content/en/docs/architecture/bugzilla.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Bugzilla Integration"
+description: An overview of the integration of Bugzilla with the Openshift CI
+---
+
+## Linking Pull Requests to Bugzilla bugs
+
+Pull requests can reference a Bugzilla bug in their title, for example
+`Bug 123: Rest of the PR title`. This makes the bugzilla plugin run a couple
+of checks:
```

Comment:
people have occasionally asked me if it's possible to link a PR to multiple bugs using this mechanism.  I believe the answer is still no, but can you mention that explicitly somewhere here?

Reply:
Done